static/index.js

```diff
@@ -103,6 +103,19 @@ var setupCoffeeCache = function(cacheDir) {
   CoffeeCache.register();
 }
 
+var isPortableMode = function() {
+  // No portable mode on non-Windows
+  if (process.platform !== 'win32') return false;
+  
+  // DevMode? Nope
+  var devMode = loadSettings.devMode || !loadSettings.resourcePath.startsWith(process.resourcesPath + path.sep);
+  if (devMode) return false;
+  
+  // Compare our EXE's path to where it would normally be in an installed app
+  var ourPath = process.execPath.toLowerCase();
+  return (ourPath.indexOf(process.env.USERPROFILE.toLowerCase()) === 0);
```

Comment:
Should this comparison include a trailing slash on `USERPROFILE`?

Currently this check will pass for the following `process.execPath`/`USERPROFILE` right?

`process.execPath = C:\Users\KevinSawicki\atom\atom.exe`
`USERPROFILE = C:\Users\Kevin`